07-data_pipelines.Rmd

```diff
@@ -157,3 +157,74 @@ Link: https://airflow.tools.alpha.mojanalytics.xyz
 You can manually start ("trigger") the pipeline with this button:
 
 ![](images/pipelines/trigger.png)
+
+
+## Test pipeline in your own Airflow sandbox
+
+You can deploy your own sandboxed version of Airflow to test Dags and docker images on.
```

Comment:
Hope it doesn't sound like I'm splitting hairs but we should try and keep the case of DAGs as "DAGs" as it is an acronym (directed acyclic graph)

Reply:
Also, maybe write it as DAG (directed acyclic graph) the first time for the people that don't know what DAG stands for.

Reply:
:+1: